solc/CommandLineInterface.cpp

```diff
@@ -790,7 +799,7 @@ void CommandLineInterface::handleCombinedJSON()
 	{
 		Json::Value contractData(Json::objectValue);
 		if (requests.count(g_strAbi))
-			contractData[g_strAbi] = dev::jsonCompactPrint(m_compiler->contractABI(contractName));
```

Comment:
I removed the change for `--abi` but the whole point of this PR was to make change to `abi, natspecdev, natspecuser`  and give the user the option to use `--pretty-json` if they dont want compact.

If `--pretty-json` is not passed the output should not be different from before.